engine/src/test/java/org/camunda/bpm/engine/test/bpmn/deployment/BpmnDeploymentTest.java

```diff
@@ -115,9 +115,8 @@ private String readInputStreamToString(InputStream inputStream) {
 
   public void FAILING_testViolateProcessDefinitionIdMaximumLength() {
     try {
-      repositoryService.createDeployment()
-        .addClasspathResource("org/camunda/bpm/engine/test/bpmn/deployment/processWithLongId.bpmn20.xml")
-        .deploy();
+      testRule.deploy(repositoryService.createDeployment()
+        .addClasspathResource("org/camunda/bpm/engine/test/bpmn/deployment/processWithLongId.bpmn20.xml"));
```

Comment:
[Optional] Should we refactor this code into an AssertJ assertion?

Reply:
👍